bedrock/etc/templates/etc/firefox/retention/thank-you-referral.html

```diff
@@ -0,0 +1,80 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "base-pebbles.html" %}
```

Comment:
If this is for 57, should the open graph image have the new logo? (When testing the Facebook link, I see it's the old/current logo.)

Reply:
Good call, I'll update.

Reply:
Oh actually it's the Firefox hub page being shared, not this page, so it'll be getting get the new logo in #5212 already.

Reply:
Ah, right. Nice!